app/manifest/v3/chrome.json

```diff
@@ -1,6 +1,6 @@
 {
   "content_security_policy": {
-    "extension_pages": "default-src 'self'; frame-ancestors 'none'"
+    "extension_pages": "script-src 'self'; object-src 'self';"
```

Comment:
I think we will want to keep `frame-ancestors 'none'`

Reply:
Exactly, was about to leave the same comment